docs/source/research/literature.md

```diff
@@ -33,37 +31,25 @@ all:
 ---
 ```
 
-## Optimization
+## Global Optimization
 
-```{bibliography} bibliography/Optimization.bib
+```{bibliography} bibliography/Bayesian_Optimization.bib
```

Comment:
```suggestion
```{bibliography} bibliography/Global_Optimization.bib
```

Reply:
@christopheroates you can either use the "Commit suggestion" button above to merge the suggested change into your fork, or edit `literature.md` manually in your fork. The proposed change simply fixes the link in the `literature.md` file to the correct bib file such the website compiles without errors. Same for the review comment below :)

Reply:
*Global* v. *Local* Optimization is fine for me, although of course the term **Bayesian Optimization** certainly has a life of its own now. So maybe it should be mentioned in the heading of that section. 

@christopheroates : For me the key difference between BO and "Optimization" is that BO collects "external" data, which is presumed to be near-arbitrarily expensive. So it's all about sample efficiency. And in fact, Bayesian Optimizers invariably call a numerical optimization routine as an inner loop. For those numerical optimizers, computational cost is a key concern. Part of the way they deal with this is that they usually consider local optima only. 

In the deep learning community (where BO is quite popular), BayesOpt is seen as the tool to optimize hyperparameters, the outer loop, while *optimizers* are used to do the actual, high-dimensional, stochastic training. 

This hopefully also explains why I want to keep these sections separate: To avoid people getting the wrong impression that "probabilistic optimizers are Bayesian optimizers, and Bayesian optimizers are super-expensive. Thus probabilistic optimizers are not relevant for deep learning".